cluster/cluster.go

```diff
@@ -165,8 +170,8 @@ func (c *Cluster) DeployWorkerPlane(ctx context.Context, svcOptionData map[strin
 		workerNodePlanMap[workerHost.Address] = BuildRKEConfigNodePlan(ctx, c, workerHost, workerHost.DockerInfo, svcOptionData)
 		if !workerHost.IsControl && !workerHost.IsEtcd {
 			workerOnlyHosts = append(workerOnlyHosts, workerHost)
-		} else {
-			multipleRolesHosts = append(multipleRolesHosts, workerHost)
+		} else if workerHost.IsEtcd {
```

Comment:
Please add comment here why we are making this differentiation for people that look at this later